.github/settings.yml

```diff
@@ -0,0 +1,18 @@
+# The settings are inherited from common repository repo-settings (same file name and location)
+# You can append settings that are deeply-merged with the inherited settings.
+#
+# When you have installed the GitHub App "repo-settings" in this repository,
+# any change of this settings.yml file is detected by the GitHub App and
+# the settings of this repository are updated immediately.
+
+_extends: repo-settings
```

Comment:
After https://github.com/terraform-ibm-modules/repo-settings/pull/9 is merged, this needs to be updated to `repo-settings:.github/common-settings.yml`

Reply:
@vburckhardt this pr is now merged, if you want to make this change to this PR since you are working on it